sys/arduino/include/arduino.hpp

```diff
@@ -18,16 +18,27 @@
  * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
  */
 
-#ifndef ARDUINO_H
-#define ARDUINO_H
+#ifndef ARDUINO_HPP
+#define ARDUINO_HPP
 
 extern "C" {
+#include <stdint.h>
 #include "periph/gpio.h"
 #include "arduino_board.h"
 }
 
 #include "serialport.hpp"
 
+/**
+ * @brief    Ardunio boolean data type definion
```

Comment:
2 typos: s/Ardunio/Arduino/ and s/definion/definition/

Here and below.

Reply:
Oh yes, typos are a big problem of mine :worried:

Reply:
Fixed.